x/dispensation/Steps-AuthorizedRunner.md

```diff
@@ -0,0 +1,200 @@
+# Steps to follow by Authorized Runner
+
+## Requirements
+1. A dispensation should already be created : A dispensation should have been created, and the authorized runner address assigned to it.
```

Comment:
1. A dispensation should have been created, and the authorized runner address assigned to it.